server/controllers/article.controller.js

```diff
@@ -204,5 +209,92 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+    * @description gets a single article
+    * @param {object} req
+    * @param {object} res
+    * @returns {object} An article and it's associated comments
+    */
+  static async getOneArticle(req, res) {
+    const { slug: articleSlug } = req.params;
+    const existingArticle = await Article.findOne({
+      where: { slug: articleSlug },
+      include: [
+        {
+          model: User,
+          as: 'author',
+          attributes: ['id', 'username', 'bio', 'image']
+        }
+      ],
+    });
```

Comment:
@TheDrizzyWay good work, I think you could use the existing middleware for this functionality, or what do you think

Reply:
I intended doing that, but the branch containing the function has not been merged, and we are all pressed for time.